codespell_lib/data/dictionary.txt

```diff
@@ -29108,6 +29108,10 @@ riminders->reminders
 riminding->reminding
 rimitives->primitives
 rininging->ringing
+rsicv->riscv
+RSICV->RISCV
+rsic-v->risc-v
+RSIC-V->RISC-V
```

Comment:
So we always lowercase the LHS, but ideally the left should be what case is appropriate. It's not used currently but it's there for the future!

Reply:
Hope I get it right this time :)